demos/twa-post-message/src/main/res/drawable/ic_launcher_background.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm going to trust you that the XML and resources look nice.

Reply:
That was the autogenerated resources, do we need to have an icon for each demo?